README.rst

```diff
@@ -43,68 +43,89 @@ SQLAlchemy Dialect for EXASOL DB
     :alt: PyPI - Downloads
 
 
-How to get started
-------------------
+Getting Started with SQLAlchemy-Exasol
+--------------------------------------
+SQLAlchemy-Exasol supports multiple dialects, primarily differentiated by whether they are ODBC or Websocket based.
 
-Currently, sqlalchemy-exasol supports multiple dialects. The core difference
-being if the dialect is :code:`odbc` or :code:`websocket` based.
+Choosing a Dialect
+++++++++++++++++++
+
+We recommend using the Websocket-based dialect due to its simplicity. ODBC-based dialects demand a thorough understanding of (Unix)ODBC, and the setup is considerably more complex.
+
+.. note::
 
-Generally, we advise to use the websocket based Dialect, because odbc
-based dialects require a good understanding of (unix)ODBC and the setup is
-significant more complicated.
+   To use an ODBC-based dialect, you must specify it as an extra during installation.
```

Comment:
should we have a quick installation guide to show how install it without odbc, because this only gets shown way later

Reply:
I think it is fine, because if you do nothing (standard install) you get it.